SwiftMessageBar/SwiftMessageBar.swift

```diff
@@ -121,9 +121,11 @@ public final class SwiftMessageBar {
   /// - Returns: A UUID for the message. Can be used to cancel the display of a specific message
   public static func showMessageWithTitle(_ title: String? = nil, message: String? = nil, type: MessageType,
                                           duration: TimeInterval = 3, dismiss: Bool = true,
-                                          callback: Callback? = nil) -> UUID {
+                                          languageDirection: NSLocale.LanguageDirection = .leftToRight,
```

Comment:
CocoaPods doesn't update the master repo per default anymore, you can refresh it by `pod repo update` or running `pod update` after changing your Podfile should normally have the same effect